FastSurferCNN/README.md

```diff
@@ -10,66 +10,77 @@ The network was trained with conformed images (UCHAR, 1-0.7 mm voxels and standa
-* --conformed_name: name of the conformed MRI_volume (the input volume is always first conformed, if not already, and the result is saved under the given name, __default: orig.mgz__)
-* --t: search tag limits processing to subjects matching the pattern (e.g. sub-* or 1030*...)
-* --sd: Path to output directory (where should predictions be saved). Will be created if it does not already exist.
-* --seg_log: name of log-file (information about processing is stored here; If not set, logs will not be saved). Saved in the same directory as the predictions.
-* --strip: strip suffix from path definition of input file to yield correct subject name. (Optional, if full path is defined for --t1)
-* --lut: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
-* --seg: Name of intermediate DL-based segmentation file (similar to aparc+aseg).
-* --cfg_cor: Path to the coronal config file
-* --cfg_sag: Path to the axial config file
-* --cfg_ax: Path to the sagittal config file
+* `--in_dir`: Path to the input volume directory (e.g /your/path/to/ADNI/fs60) or 
+* `--csv_file`: Path to csv-file listing input volume directories
+* `--t1`: name of the T1-weighted MRI_volume (like mri_volume.mgz, __default: orig.mgz__)
+* `--conformed_name`: name of the conformed MRI_volume (the input volume is always first conformed, if not already, and the result is saved under the given name, __default: orig.mgz__)
+* `--t`: search tag limits processing to subjects matching the pattern (e.g. sub-* or 1030*...)
+* `--sd`: Path to output directory (where should predictions be saved). Will be created if it does not already exist.
+* `--seg_log`: name of log-file (information about processing is stored here; If not set, logs will not be saved). Saved in the same directory as the predictions.
+* `--strip`: strip suffix from path definition of input file to yield correct subject name. (Optional, if full path is defined for `--t1`)
+* `--lut`: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
+* `--seg`: Name of intermediate DL-based segmentation file (similar to aparc+aseg).
+* `--cfg_cor`: Path to the coronal config file
+* `--cfg_sag`: Path to the axial config file
+* `--cfg_ax`: Path to the sagittal config file
 
 #### Checkpoints
-* --ckpt_sag: path to sagittal network checkpoint
-* --ckpt_cor: path to coronal network checkpoint
-* --ckpt_ax: path to axial network checkpoint
+* `--ckpt_sag`: path to sagittal network checkpoint
+* `--ckpt_cor`: path to coronal network checkpoint
+* `--ckpt_ax`: path to axial network checkpoint
 
 #### Optional commands
-* --clean: clean up segmentation after running it (optional)
-* --run_viewagg_on: Define where the view aggregation should be run on. 
+* `--clean`: clean up segmentation after running it (optional)
+* `--viewagg_device`: Define where the view aggregation should be run on. 
```

Comment:
--viewagg_device <torch-compatible-device> -- this can be cpu, cuda, cuda:<device_num>, etc.

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)